webapp/src/components/dashboard.tsx

```diff
@@ -26,7 +27,7 @@ export function Dashboard() {
   const data = [
     {
       key: Locale.Dashboard.StartTime,
-      value: distributionDetail?.startTime
+      value: distributionDetail && zonedDateFormatYYYYMMddHHMMSS(distributionDetail.startTime)
```

Comment:
I know this fixes the linked bug .. but are there other fields where this would be needed as well .. like endtime or whatever that appear in the UI?